trapi.rkt

```diff
@@ -330,11 +332,30 @@
   (summary-fragment-nodes (condensed-summary-fragment cs)))
 (define (condensed-summary-edges cs)
   (summary-fragment-edges (condensed-summary-fragment cs)))
+(define (condensed-summary-scores cs)
+  (summary-fragment-scores (condensed-summary-fragment cs)))
 (define (path->key path)
   (string->symbol (number->string (equal-hash-code path))))
 
 (define (merge-summary-attrs r s)
-  (map (lambda (r-attr s-attr) (append r-attr s-attr)) r s))
+  (define (merge-lists r s accessor)
+    (append (accessor r) (accessor s)))
+
+  (define (merge-scores r s)
+    (define current-scores (summary-fragment-scores s))
+    (define new-score (car (hash->list (summary-fragment-scores r))))
```

Comment:
Is it intentional that you only take the first?  What are you doing with the rest of them?

Reply:
There can only be one score per result.